google/oauth2/service_account.py

```diff
@@ -192,7 +200,12 @@ def __init__(
             self._additional_claims = additional_claims
         else:
             self._additional_claims = {}
-        self._trust_boundary = {"locations": [], "encoded_locations": "0x0"}
+        self._trust_boundary_enabled = (
+            os.environ.get(TRUST_BOUNDARY_LOOKUP_ENABLED_ENV) == "1"
```

Comment:
This is inconsistent with https://google.aip.dev/auth/4110, where "true|false" is used

My suggestion:

Define this in an AIP.

Check for the presence of the ENV variable to enable it. It can be unset to "toggle" off.

Reply:
Only for presents or not? I thought when we explicitly set some variable, it should be exactly match a value we want. like `true`?

Reply:
By the way. This is just a temporary solution for the phase since the backend is not finished on lookup endpoint.
Once they are done, we should go through this flow any way.

Reply:
@westarle any opinions on this? 

What is the justification for the temporary environment variable?

Reply:
Hi @clundin25 , we decided to get rid of a temporary control since any release of SDK code won't get back. For this change, we just avoid this part of the logic because the endpoint is not ready yet.